src/content/h1/info/readme.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hard-coded Data
+keywords:
+  - color
+  - data
+  - engine
+  - hard-coded
+  - info
+thanks:
+  - to: MosesOfEgypt
+    for: Armor color values
+  - to: Lavadeeto
+    for: Armor color testing and picture
+---
+There are lots of hard-coded values in Halo's engine, such as the multiplayer
+armor color. These values do not typically appear anywhere in tag data, and
+cannot be directly changed or controlled.
+
+# Multiplayer Armor Colors
+
+<figure>
+  <a href="armor_colors.png">
+    <img src="armor_colors.png" alt="Armor colors in order"/>
+  </a>
+  <figcaption>
+    Spartan armor colors with shaders as they appear ingame.
+    The far-left is the single-player color.
+  </figcaption>
```

Comment:
The `figcaption` element is a flow root, meaning it's like a container for a flow of content (e.g. paragraphs, images, headings), although we'd almost always just have a single paragraph in it. The text inside it should be wrapped with `<p></p>` tags.